cartography/intel/aws/cloudwatch.py

```diff
@@ -75,6 +75,192 @@ def cleanup_event_buses(neo4j_session: neo4j.Session, common_job_parameters: Dic
     run_cleanup_job('aws_import_cloudwatch_event_buses_cleanup.json', neo4j_session, common_job_parameters)
 
 
+@timeit
+@aws_handle_regions
+def get_event_rules(boto3_session: boto3.session.Session, region):
+    event_rules = []
+    try:
+        client = boto3_session.client('events', region_name=region)
+        paginator = client.get_paginator('list_rules')
+
+        page_iterator = paginator.paginate()
+        for page in page_iterator:
+            event_rules.extend(page['Rules'])
+
+        for event_rule in event_rules:
+            event_rule['region'] = region
+
+    except ClientError as e:
+        logger.error(f'Failed to call CloudWatch event list_rules: {region} - {e}')
+    return event_rules
+
+
+def load_event_rules(session: neo4j.Session, event_buses: List[Dict], current_aws_account_id: str, aws_update_tag: int) -> None:
+    session.write_transaction(_load_event_rules_tx, event_buses, current_aws_account_id, aws_update_tag)
+
+
+@timeit
+def _load_event_rules_tx(tx: neo4j.Transaction, event_buses: List[Dict], current_aws_account_id: str, aws_update_tag: int) -> None:
+    query: str = """
+    UNWIND {Records} as record
+    MERGE (rule:AWSCloudWatchEventRule{id: record.Arn})
+    ON CREATE SET rule.firstseen = timestamp(),
+        rule.arn = record.Arn
+    SET rule.lastupdated = {aws_update_tag},
+        rule.name = record.Name,
+        rule.region = record.region,
+        rule.event_bus_name = record.EventBusName,
```

Comment:
we should also fetch & load event buses from event bridge api and build relationship with the event rule in a HAS relationship.

Reference: https://docs.aws.amazon.com/cli/latest/reference/events/list-event-buses.html